WebDevStudios/wd_s
# Feature/depedency bump

Closes #570 #576 #577 #578

### COREY! MERGE THIS BEFORE #573 

### DESCRIPTION

- I went through each NPM package one-by-one and tested after an upgrade
- The following package broke on upgrade (so I left it alone):
  - `copy-webpack-plugin`
- I did not attempt to upgrade Husky and Lint-Staged in anticipation of #571 

### STEPS TO VERIFY

- Check out this branch
- Remove node_modules `rm -rf node_modules`
- Run `npm i`
- Run `npm run start`
